zebra-chain/src/types.rs

```diff
@@ -81,8 +86,26 @@ impl ZcashDeserialize for LockTime {
     }
 }
 
+#[cfg(test)]
+impl Arbitrary for LockTime {
+    type Parameters = ();
+
+    fn arbitrary_with(_args: ()) -> Self::Strategy {
+        prop_oneof![
+            (0u32..500_000_000_u32).prop_map(|n| LockTime::Height(BlockHeight(n))),
+            Just(LockTime::Time(
+                Utc.timestamp(Utc::now().timestamp() as i64, 0)
```

Comment:
Do we want to generate random timestamps with `prop_map` here? I'm not sure how tests would be reproducible with `Utc::now()`.